public/newInstanceModal.js

```diff
@@ -47,11 +56,11 @@ angular.module('broccoli')
     vm.cancel = cancel;
 
     function ok() {
-      Object.keys(vm.paramsToValue).forEach(function(param) {
+        Object.keys(vm.paramsToValue).forEach(function(param) {
```

Comment:
Let's not change the formatting in the same commit :)

I recommend you look at your changes before you commit them @IshmeetKaur (e.g. with a GUI or `git add -p`).